src/utils.js

```diff
@@ -46,15 +46,23 @@ const writeFile = ({ filePath, content }) => new Promise((resolve, reject) => {
  * Sets up a basic HTML page to apply the content to
  * @return {xmlbuilder}
  */
-const createHtmlBase = ({ pageTitle, stylesheet }) => xmlbuilder.create({
-	html: {
-		head: {
-			meta: { '@charset': 'utf-8' },
-			title: { '#text': pageTitle },
-			style: { '@type': 'text/css', '#text': stylesheet },
+const createHtmlBase = ({ pageTitle, stylesheet, useCssFile }) => {
```

Comment:
Replace the 'useCssFile' with something like 'stylesheetPath' to differentiate between the stylesheet-content and the path to it.